Assets/Scripts/Player/RevolverBehaviour.cs

```diff
@@ -8,27 +8,39 @@ public class RevolverBehaviour : MonoBehaviour
     private ComboBehaviour thisCombo;
     CharacterBehaviour thisPlayer;
 
-     public List<GameObject> bullets;
+    public List<GameObject> bullets;
     public List<GameObject> bulletPrefabs;
     public Transform bulletspawn,crouchingBulletSpawn;
     GameObject desiredBullet;
 
-    public bool hasTemporaryBullet;
+    public int hasTemporaryBullet;
         
     //VISUAL
     public float bulletSpeed = 60;
 
     //TARGETABLE REFERS TO: WHAT WILL THE PLAYER'S BULLET COLLIDE WITH - WALLS, ENTITIES ETC.
     public LayerMask excludeTarget;
 
+    //UI ELEMENTS
+    private uiRubiBehaviour thisRubiBehaviour;
+
     void Awake()
     {
         thisCombo = GetComponent<ComboBehaviour>();
         thisPlayer = GetComponent<CharacterBehaviour>();
 
-        hasTemporaryBullet = false;
-    }
+        hasTemporaryBullet = 0;
 
+        thisRubiBehaviour = FindObjectOfType<uiRubiBehaviour>();
+    }
+    private void Start()
+    {
+        //INITIALISE BULLETS INTO RUBI
+        for (int i=0; i <6; i++)
```

Comment:
nit: `for (int i = 0; i < 6; i++)` is prettier

Reply:
fixed